corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -163,7 +166,15 @@ function FreeTextEntry(question, options) {
     };
 
     self.helpText = function() {
-        return isPassword ? 'Password' : 'Free response';
+        if (isPassword) {
+            return gettext('Password');
+        }
+        switch (self.datatype) {
+        case Formplayer.Const.BARCODE:
```

Comment:
yeah it's the way dimagimon likes it (that's why it yelled at the one above, that switch statement was written before we had JS lint)